theorems/T000010.md

```diff
@@ -7,8 +7,9 @@ if:
 then:
   P000052: true
 converse:
-- T000143
 - T000044
+- T000085
+- T000077
```

Comment:
FWIW, this was one where we looked to have deleted `T143` because it was already provable by `T85 + T77`. 

I wonder if we should try to split theorems up in general ... e.g. should `second countable => first countable + separable + lindelof` actually just be three separate theorems?

Reply:
Hmm. So what can we model if `if` is a list of conjuncted properites, and `then` is a single property?

```
P^Q=>R directly
PvQ=>R rewrites to P=>R and Q=>R
P=>Q^R rewrites to P=>Q and P=>R
P=>QvR rewrites to P^~Q=>R (or P^~R=>Q)
```

I like it -- this seems saner than trying to support really elaborate logical implications `(PvQ)^R=>AvBvC` can be modeled by `P^R^~A^~B=>C` and `Q^R^~A^~B=>C`.

Reply:
Devil's advocate: there are some really nice characterizations like `T_3 + second-countable <=> metrizable + Lindelof` that would be split into four separate theorems. Of course, these are already two theorems.